src/OrchardCore.Modules/OrchardCore.Users/Views/Account/Login.cshtml

```diff
@@ -37,8 +37,12 @@
                             <a asp-controller="ResetPassword" asp-action="ForgotPassword" class="label-link">@T["Forgot your password?"]</a>
                         }
                     </label>
-
-                    <input asp-for="Password" class="form-control" tabindex="2" />
+                    <div class="input-group">
+                        <input asp-for="Password" class="form-control" tabindex="2" />
+                        <div class="input-group-append">
+                            <button class="btn btn-outline-secondary" type="button" id="togglePassword"><i class="icon far fa-eye"></i></button>
```

Comment:
Requires font-awesome.

Reply:
Yes

Reply:
Then require the Resource or else if the theme doesn't have it, it won't work.

Reply:
done